src/resources/views/crud/fields/fetch.blade.php

```diff
@@ -0,0 +1,246 @@
+@php
+
+    //in case entity is superNews we want the url friendly super-news
+    $routeEntity = strtolower(preg_replace('/([a-zA-Z])(?=[A-Z])/', '$1-', $field['entity']));
+
+    $connected_entity = new $field['model'];
+
+    $multiple = $field['multiple'] ?? $crud->relationAllowsMultiple($field['relation_type']);
```

Comment:
I think we should have a single source of truth here too.

**The problem:** You define a variable called ```$multiple``` starting from ```$field['multiple']```. Then below, in the HTML, you use ```$multiple``` once, but also ```$field['multiple']``` once. I'm pretty sure that's a mistake, and causes some sort of error. If we had a single source of truth for that information, that couldn't have happened.

---

**I propose we use ```$field``` as the single source of truth in this file.** Rather than creating new variables, we can overwrite an attribute of the ```$field``` array. In this case, instead of creating ```$multiple```, we could just overwrite ```$field['multiple']```. That way, it's easy to follow where that information is.

We can do the same thing for:
- placeholder
- allows_null
- current_value
- etc

I think it would be much easier to understand, like you've _already_ done for ```data-source```:
```php
    $field['data_source'] = $field['data_source'] ?? url($crud->route.'/fetch/'.$routeEntity);
```
Even if we set a default for that one, we use a single source of truth for that bit of information: ```$field['data_source']``` - it will never be confusing where to find that information.